src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -982,7 +982,15 @@ Namespace Microsoft.CodeAnalysis.Semantics
         Private Function CreateBoundBlockOperation(boundBlock As BoundBlock) As IBlockStatement
             Dim statements As Lazy(Of ImmutableArray(Of IOperation)) = New Lazy(Of ImmutableArray(Of IOperation))(
                 Function()
-                    Return boundBlock.Statements.Select(Function(n) Create(n)).Where(Function(s) s.Kind <> OperationKind.None).ToImmutableArray()
+                    Return boundBlock.Statements.Where(
+                            Function(statement)
+                                ' Don't include the method/operator/accessor declaration in the block
+                                Return statement.Syntax.Kind() <> SyntaxKind.OperatorStatement AndAlso
+                                       statement.Syntax.Kind() <> SyntaxKind.FunctionStatement AndAlso
+                                       statement.Syntax.Kind() <> SyntaxKind.GetAccessorStatement AndAlso
+                                       statement.Syntax.Kind() <> SyntaxKind.SetAccessorStatement
```

Comment:
The issue here is that, when we move `fixed` statements to be NoneOperation, they are then not included in any block statements, which means that analyzers will miss both the fixed statement and any children it declares. This is removing the filtering, but at the same time that introduces these first-line bound nodes are included in the block. We have tests for the 3 event blocks, and those were unaffected by this change.